docs/querying/basics.md

```diff
@@ -242,7 +242,16 @@ These 2 queries will produce the same result.
 
 This modifier is disabled by default since it breaks the invariant that PromQL
 does not look ahead of the evaluation time for samples. It can be enabled by setting
-`--enable-feature=promql-at-modifier` flag. It will be enabled by default in the future.
+`--enable-feature=promql-at-modifier` flag. See [disabled features](../disabled_features.md) for more about this flag.
```

Comment:
"… for more details about …" (I like that more. Currently feels like something is missing.)